draft-ietf-rtcweb-jsep.xml

```diff
@@ -3389,6 +3411,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           the end-of-candidates indication as described in
           <xref target="I-D.ietf-ice-trickle" /> Section 11.</t>
 
+          <t>If the dtls-id has changed from the previous remote description,
```

Comment:
Looking at this more closely, I think you probably want to move this text to the 'applying an answer' section, which already has a para on when to create a DTLS connection.